meta-refkit-extra/recipes-multimedia/btspeaker/btspeaker.bb

```diff
@@ -0,0 +1,90 @@
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${META_REFKIT_BASE}/COPYING.MIT;md5=838c366f69b72c5df05c96dff79b35f2"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+SRC_URI = "\
+        file://btspeaker.py \
+        file://btspeaker.service \
+        file://client.conf \
+        file://leds.rules \
+        file://btspeaker-bluetooth.conf \
+        file://btspeaker-connman.conf \
+        file://pulseaudio-bluetooth.conf \
+        file://audio.conf \
+        file://buttons.asl \
+        file://leds.asl \
+        file://btpairing.wav \
+        file://btstartup.wav \
+        file://btsuccess.wav \
+"
+
+DEPENDS = "acpica-native"
+
+RDEPENDS_${PN} = "\
+        python-dbus \
+        python-pygobject \
+        python-pyalsaaudio\
+        python-evdev \
+"
+
+inherit useradd deploy systemd
+
+USERADD_PACKAGES = "${PN}"
+GROUPADD_PARAM_${PN} = "-g 1100 -r leds"
+USERADD_PARAM_${PN} = "-u 1200 -d /home/btspeaker -r -s /bin/bash -G audio,leds,input btspeaker"
+SYSTEMD_SERVICE_${PN} = "btspeaker.service"
+
+do_install () {
+        install -d -m 755 ${D}/home/btspeaker
+        install -d -m 755 ${D}/home/btspeaker/.pulse
+        install -d -m 755 ${D}${sysconfdir}/udev/rules.d/
+        install -d -m 755 ${D}${sysconfdir}/dbus-1/system.d/
+        install -d -m 755 ${D}${sysconfdir}/bluetooth/
+        install -d -m 755 ${D}${systemd_system_unitdir}
+
+        install -p -m 644 ${WORKDIR}/btspeaker.py ${D}/home/btspeaker/
+        install -p -m 644 ${WORKDIR}/client.conf ${D}/home/btspeaker/.pulse/
+        install -p -m 644 ${WORKDIR}/btpairing.wav ${D}/home/btspeaker/
+        install -p -m 644 ${WORKDIR}/btstartup.wav ${D}/home/btspeaker/
+        install -p -m 644 ${WORKDIR}/btsuccess.wav ${D}/home/btspeaker/
+        install -p -m 644 ${WORKDIR}/leds.rules ${D}${sysconfdir}/udev/rules.d/
+        install -p -m 644 ${WORKDIR}/btspeaker-bluetooth.conf ${D}${sysconfdir}/dbus-1/system.d/
+        install -p -m 644 ${WORKDIR}/btspeaker-connman.conf ${D}${sysconfdir}/dbus-1/system.d/
+        install -p -m 644 ${WORKDIR}/pulseaudio-bluetooth.conf ${D}${sysconfdir}/dbus-1/system.d/
+        install -p -m 644 ${WORKDIR}/audio.conf ${D}${sysconfdir}/bluetooth/
+
+        install -p -m 644 ${WORKDIR}/btspeaker.service ${D}${systemd_system_unitdir}
+
+        chown -R btspeaker ${D}/home/btspeaker
+}
+
+do_compile() {
+        rm -fr ${WORKDIR}/acpi-tables/kernel
+        install -d ${WORKDIR}/acpi-tables/kernel/firmware/acpi
+
+        iasl -p ${WORKDIR}/acpi-tables/kernel/firmware/acpi/buttons.asl ${WORKDIR}/buttons.asl
+        iasl -p ${WORKDIR}/acpi-tables/kernel/firmware/acpi/leds.asl ${WORKDIR}/leds.asl
+}
+
+do_deploy() {
+        cd ${WORKDIR}/acpi-tables
+        find kernel | cpio -H newc -o > ${DEPLOYDIR}/acpi-tables.cpio
+}
+addtask deploy before do_build after do_compile
+
+FILES_${PN} = " \
+        /home/btspeaker/btspeaker.py \
+        /home/btspeaker/.pulse/client.conf \
+        /home/btspeaker/btpairing.wav \
```

Comment:
btspeaker.py and all *.wav are from me originally and would go with refkit MIT. Actually I have to think about the pulse client.conf because it is a modified copy from pulse... hmm I may have to do it somehow else. good catch.